pkg/skaffold/errors/errors.go

```diff
@@ -21,14 +21,15 @@ import (
 )
 
 const (
-	Build       = phase("Build")
-	Deploy      = phase("Deploy")
-	StatusCheck = phase("StatusCheck")
-	FileSync    = phase("FileSync")
+	Build       = Phase("Build")
+	Deploy      = Phase("Deploy")
+	StatusCheck = Phase("StatusCheck")
+	FileSync    = Phase("FileSync")
+	Dev         = Phase("Dev")
```

Comment:
Is `Dev` really a phase?  (We don't actually use Dev from what I can tell?)

Reply:
it is not a phase, but dev-init is a stage or step in dev loop where errors can happen. 
I wanted to capture this "init" step errors.

Shd "DevInit" be better?

Reply:
also, i can rename "Phase" to "Stage"

Reply:
Change Dev -> DevInit
Add TearDown
Add comment saying these are phases in dev loop.